src/frontend/admin/src/components/TickerTableRow.js

```diff
@@ -41,6 +41,14 @@ export function TickerTableRow({ data, onShow, onEdit, onDelete }) {
                 {data.settings.type === "image" && <ImageIcon/>}
             </TableCell>
             <TableCell component="th" scope="row">
+                {data.settings.type === "clock" &&
+                    (editData === undefined ? data.settings.timeZone : (
+                        <Box onSubmit={onSubmitEdit} component="form" type="submit">
+                            <TextField autoFocus hiddenLabel fullWidth defaultValue={data.settings.text}
+                                id="filled-hidden-label-small" type="text" size="small" sx={{ width: 1 }}
+                                onChange={onChangeFieldEventHandler(setEditData, "timeZone")}/>
```

Comment:
This probably needs vaidation

Reply:
fix defaultValue

Reply:
I mean validate user inputed text before submitting the form

Reply:
Fixed